python/lsst/sims/alertsim/catalogs/diaObjectCommons.py

```diff
@@ -1,179 +1,401 @@
+	return array_to_dict(cols, vals)
+
+    def get_rLcPeriodic(self):
+	vals = self.randomFloatArr(32, -1)
+	cols = ['rLcPeriodic01', 'rLcPeriodic02', 'rLcPeriodic03', 'rLcPeriodic04', 'rLcPeriodic05', 'rLcPeriodic06', 'rLcPeriodic07', 'rLcPeriodic08', 'rLcPeriodic09', 'rLcPeriodic10', 'rLcPeriodic11', 'rLcPeriodic12', 'rLcPeriodic13', 'rLcPeriodic14', 'rLcPeriodic15', 'rLcPeriodic16', 'rLcPeriodic17', 'rLcPeriodic18', 'rLcPeriodic19', 'rLcPeriodic20', 'rLcPeriodic21', 'rLcPeriodic22', 'rLcPeriodic23', 'rLcPeriodic24', 'rLcPeriodic25', 'rLcPeriodic26', 'rLcPeriodic27', 'rLcPeriodic28', 'rLcPeriodic29', 'rLcPeriodic30', 'rLcPeriodic31', 'rLcPeriodic32']
+	return array_to_dict(cols, vals)
+
+    def get_iLcPeriodic(self):
+	vals = self.randomFloatArr(32, -1)
+	cols = ['iLcPeriodic01', 'iLcPeriodic02', 'iLcPeriodic03', 'iLcPeriodic04', 'iLcPeriodic05', 'iLcPeriodic06', 'iLcPeriodic07', 'iLcPeriodic08', 'iLcPeriodic09', 'iLcPeriodic10', 'iLcPeriodic11', 'iLcPeriodic12', 'iLcPeriodic13', 'iLcPeriodic14', 'iLcPeriodic15', 'iLcPeriodic16', 'iLcPeriodic17', 'iLcPeriodic18', 'iLcPeriodic19', 'iLcPeriodic20', 'iLcPeriodic21', 'iLcPeriodic22', 'iLcPeriodic23', 'iLcPeriodic24', 'iLcPeriodic25', 'iLcPeriodic26', 'iLcPeriodic27', 'iLcPeriodic28', 'iLcPeriodic29', 'iLcPeriodic30', 'iLcPeriodic31', 'iLcPeriodic32']
+	return array_to_dict(cols, vals)
+
+    def get_zLcPeriodic(self):
+	vals = self.randomFloatArr(32, -1)
+	cols = ['zLcPeriodic01', 'zLcPeriodic02', 'zLcPeriodic03', 'zLcPeriodic04', 'zLcPeriodic05', 'zLcPeriodic06', 'zLcPeriodic07', 'zLcPeriodic08', 'zLcPeriodic09', 'zLcPeriodic10', 'zLcPeriodic11', 'zLcPeriodic12', 'zLcPeriodic13', 'zLcPeriodic14', 'zLcPeriodic15', 'zLcPeriodic16', 'zLcPeriodic17', 'zLcPeriodic18', 'zLcPeriodic19', 'zLcPeriodic20', 'zLcPeriodic21', 'zLcPeriodic22', 'zLcPeriodic23', 'zLcPeriodic24', 'zLcPeriodic25', 'zLcPeriodic26', 'zLcPeriodic27', 'zLcPeriodic28', 'zLcPeriodic29', 'zLcPeriodic30', 'zLcPeriodic31', 'zLcPeriodic32']
+	return array_to_dict(cols, vals)
+
+    def get_yLcPeriodic(self):
+	vals = self.randomFloatArr(32, -1)
+	cols = ['yLcPeriodic01', 'yLcPeriodic02', 'yLcPeriodic03', 'yLcPeriodic04', 'yLcPeriodic05', 'yLcPeriodic06', 'yLcPeriodic07', 'yLcPeriodic08', 'yLcPeriodic09', 'yLcPeriodic10', 'yLcPeriodic11', 'yLcPeriodic12', 'yLcPeriodic13', 'yLcPeriodic14', 'yLcPeriodic15', 'yLcPeriodic16', 'yLcPeriodic17', 'yLcPeriodic18', 'yLcPeriodic19', 'yLcPeriodic20', 'yLcPeriodic21', 'yLcPeriodic22', 'yLcPeriodic23', 'yLcPeriodic24', 'yLcPeriodic25', 'yLcPeriodic26', 'yLcPeriodic27', 'yLcPeriodic28', 'yLcPeriodic29', 'yLcPeriodic30', 'yLcPeriodic31', 'yLcPeriodic32']
+	return array_to_dict(cols, vals)
+
+    def get_uLcNonPeriodic(self):
+	vals = self.randomFloatArr(32, -1)
```

Comment:
I think the first argument of `randomFloatArr` should be 20, not 32

Reply:
sure, for all of them